frontend/src/js/drawCircles.js

```diff
@@ -31,11 +31,14 @@
                               hmdaStat(data), style );
 
         //  We will use the geoid when redrawing
-        circle.geoid = geo.geoid;
+        circle.geoid = geo.geoid || '';
         circle.volume = geo.volume;
         circle.type = "tract-circle";
         circle.keyCircle = 0;
 
+        // Trim the year from the census ID
+        circle.geoid = circle.geoid.replace(new RegExp('^' + selectedYear), '');
+
```

Comment:
urgh we should avoid parsing the ID for content if possible. I think all geoids because the ID should be meaningless. I think all tracts are state+county+tract from the geo table. Is it easy to grab those?